js/quote_tool.js

```diff
@@ -1 +1,58 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+// Qouting tool logic should be placed here.
+const PRICENIGTH = 55000;
+const BUSTRANS = 75000;
+const BOATTRANS = 75000;
+// Tour aguacate, playa soledad
+const TOURAPS = 65000;
+// Tour sazurro, la miel
+const TOURSLM = 60000;
+// Hike el cielo, el trebol
+const HIKEECET = 40000;
+// Hike el paraiso
+const HIKEP = 30000;
+//Hike la coquerita
+const HIKECO = 5000;
+// guide hike la coquerita
+const HIKEGUI = 10000;
+const INSURANCE = 3500;
+
+function quote(){
+    let total = 0;
+    let peopleAndNigth;
+    if (form.numberOfPeople > 0){
```

Comment:
This validation is not clear. The form should validate before sending the information to quote, you should expect that the information has been already validated.